docs-source/content/userguide/introduction/introduction.md

```diff
@@ -16,16 +16,18 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 
 ### Operator prerequisites
 
-* Kubernetes 1.13.5+, 1.14.8+, and 1.15.7+  (check with `kubectl version`).
-* Flannel networking v0.9.1-amd64 (check with `docker images | grep flannel`) *or* OpenShift SDN on OpenShift 4.3 systems.
-* Docker 18.9.1 (check with `docker version`) *or* CRI-O 1.14.7 (check with `crictl version | grep RuntimeVersion`).
+* Kubernetes 1.13.5+, 1.14.8+, and 1.15.7+  (check with `kubectl version`).  Not 1.16 or 1.17, see note below.
+  See note below for OpenShift.
```

Comment:
"See note below for OpenShift." Should this be a separate bulleted item?

Reply:
i don't think so

Reply:
So, you want this bulleted item to state, "Not supported on Kubernetes 1.16 or 1.17, see note below. See note below for OpenShift."

Also, Not supported on Kubernetes 1.16 or 1.17, see note below. -> Not supported on Kubernetes 1.16 or 1.17; see note below. (semi-colon to separate two complete sentences)

Reply:
fixed